getPrivateKeys.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# Install json parser
+apk update
+apk add jq
+
+# Read the JSON file
+json_content=$(cat anvil-config.txt)
+
+# Extract private_keys array values using jq
+private_keys=$(echo "$json_content" | jq -r '.private_keys[]')
+
+# Write private keys to a new file for easier indexing
+echo "Writing private keys to file"
+echo "$private_keys" > /shared/private-keys.txt
```

Comment:
we can avoid also this private-keys files. anvil-config has everything we need, no need IMHO to have this intermediate file.

Reply:
removed in latest commit https://github.com/waku-org/waku-simulator/pull/57/commits/71882e78b56af87b10bbba8d5f2cd3986689aff0